packages/sourcecred/src/core/ledger/ledger.js

```diff
@@ -781,8 +870,16 @@ export class Ledger {
     return this;
   }
 
+  _enableGrainIntegrationTracking(_: EnableGrainIntegration) {
```

Comment:
Breaks the pattern of the public and private function names matching
```suggestion
  _enableIntegrationTracking(_: EnableGrainIntegration) {
```